ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -238,7 +238,7 @@ public QueryMetadata getQueryExecutionPlan(final Query query) throws Exception {
   }
 
 
-  private List<Pair<String, Statement>> parseQueries(
+  public List<Pair<String, Statement>> parseQueries(
```

Comment:
This can be package private. Should also add a `// Visible for Testing` comment.